### PR TITLE
replace -v with  --suggest for oc status

### PR DESCRIPTION
**Pull request by zhouying7780**
@kasturinarra @liangxia Could you please help review , thanks . 

I've tested on my localhost and pass.

### CODE REVIEW

**Comment by zhouying7780** [comment]
And the project should be empty                                                                              # features/step_definitions/project.rb:234
      [08:40:25] INFO> Shell Commands: oc status --suggest --kubeconfig=/root/workdir/dhcp-140-138-root/ocp4_testuser-19.kubeconfig -n v1p91
      In project v1p91 on server https://api.qeci-4479-debug.qe.devcluster.openshift.com:6443
      
      pod/tolerationseconds-1 runs docker.io/ocpqe/hello-pod:latest
      
      You have no services, deployment configs, or build configs.
      Run 'oc new-app' to create an application.
      
      [08:40:28] INFO> Exit Status: 0

File: lib/openshift/project.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@ def visible?(user: nil, result: {}, quiet: false)
     alias exists? visible?
 
     def empty?(user: nil)
-      res = default_user(user).cli_exec(:status, v: true, n: name)
+      res = default_user(user).cli_exec(:status, suggest: true, n: name)
```
Comment (pruan-rht) [inline lib/openshift/project.rb:34]: @zhouying7780 how far back does `suggest` exist in the OCP release?

Comment (liangxia) [reply lib/openshift/project.rb:34]: @pruan-rht  I've checked 4.1 and 4.4, both have suggest.

**Pull request merged**